deploy/scripts/src/cri-kiwi/requestGenerator/ipvrReqGen.ts

```diff
@@ -0,0 +1,89 @@
+import { type AUTH_IPV_AUTHORISATION_REQUESTED, type F2F_YOTI_START, type IPV_F2F_CRI_VC_CONSUMED } from '../requestGenerator/ipvrReqFormat'
+import { uuidv4 } from '../../common/utils/jslib/index'
+
+const userID = uuidv4()
+const signinJourneyID = uuidv4()
+const userURN = `urn:fdc:gov.uk:2022:${userID}`
+const randomNum = Math.floor(Math.random() * 9999999) // Random sleep between 2-4 seconds
+const randomEmail = `perfSPOT${randomNum}@digital.cabinet-office.gov.uk`
+
+export function generateAuthRequest (): AUTH_IPV_AUTHORISATION_REQUESTED {
+  const timestamp = new Date().toISOString()
+  const sampleAuthRequest: AUTH_IPV_AUTHORISATION_REQUESTED = {
+    event_id: uuidv4(),
+    client_id: uuidv4(),
+    clientLandingPageUrl: 'https://www.gov.uk/request-copy-criminal-record',
+    event_name: 'AUTH_IPV_AUTHORISATION_REQUESTED',
+    rp_name: 'replay',
+    timestamp: Date.now().toString().substring(0, 10),
```

Comment:
```suggestion
    timestamp: Math.floor(Date.now()/1000).toString(),
```
I think technically this shouldn't presume the length of the unix timestamp, but this current solution will work till the year 2286 - so it's probably fine

Reply:
Updated